packages/react-reconciler/src/ReactFiberLane.js

```diff
@@ -534,6 +589,18 @@ export function getHighestPriorityLane(lanes: Lanes): Lane {
   return lanes & -lanes;
 }
 
+function getLanesOfEqualOrHigherPriority(
+  lanes: Lanes,
+  priorityLane: Lane | Lanes,
+): Lanes {
+  // Create a mask with all bits to the left or same as the highest bit of B set
+  // So if priorityLane is 0b10000, the mask would be 0b11111.
+  // If priorityLane is 0b10100, the mask would be 0b10111
+  const lowestPriorityLaneIndex = 31 - clz32(priorityLane);
+  const mask = (1 << (lowestPriorityLaneIndex + 1)) - 1;
```

Comment:
```suggestion
  // So if priorityLane is 0b10000, the mask would be 0b11111.
  // If priorityLane is 0b10100, the mask would also be 0b11111 since both have the same highest bit.
  const lowestPriorityLaneIndex = 31 - clz32(priorityLane);
  const mask = (1 << (lowestPriorityLaneIndex + 1)) - 1;
```

Both examples have the same highest bit so the mask stays the same.

Right now `getLanesOfEqualOrHigherPriority(0b11100, 0b10100)` would return `0b11100` because the mask is `0b11111` not `0b10111`

Reply:
```js
const highestPriorityLaneIndex = 31 - Math.clz32(priorityLane & -priorityLane);
const mask = ((1 << (highestPriorityLaneIndex + 1)) - 1) | priorityLane;
```
would produce the mask explained by the comment I believe.

Reply:
> Right now getLanesOfEqualOrHigherPriority(0b11100, 0b10100) would return 0b11100 because the mask is 0b11111 not 0b10111

This is the intent, I think the comment I wrote was just a typo.

I meant it to say something like:

> If priorityLane is 0b101, the mask would be 0b111.

Then you apply the mask to the first argument (`lanes`) to get the final result.

Reply:
I think it'd be clearer if I made this function return the mask itself (like the name implies) and then caller applies it, so that's what I'll do.